samples/v1.3/Elements/Action.Submit.AssociatedInputs.json

```diff
@@ -0,0 +1,25 @@
+{
+  "type": "AdaptiveCard",
```

Comment:
you'll need to reconcile the whitespace in this version (spaces) vs. the .NET version (tabs) :)